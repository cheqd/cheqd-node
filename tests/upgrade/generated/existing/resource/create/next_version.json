{
    "collectionId": "97096fb1-63ea-4f81-b296-8eb5da6ab6eb",
    "id": "7f2589aa-1279-4bd7-b704-f868772b460a",
    "name": "Test Resource Next Version",
    "resourceType": "TestResourceType",
    "data": "eyAibWVzc2FnZSI6ICJoZWxsbyB3b3JsZCJ9"
}