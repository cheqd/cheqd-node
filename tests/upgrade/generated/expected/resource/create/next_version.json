{
    "collectionId": "97096fb1-63ea-4f81-b296-8eb5da6ab6eb",
    "id": "e6a69483-a1b1-42fa-b32a-55724a341d06",
    "name": "Test Resource",
    "resourceType": "TestResourceType",
    "mediaType": "application/json",
    "created": "2022-11-01T10:54:01Z",
    "checksum": "T0ebujH3GFnZlDZRU4yN8hGYBOzrf1ig3iQXif8hSvQ=",
    "previousVersionId": null,
    "nextVersionId": null
}